docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+===================
+JDBC connector base
+===================
+
+A number of connectors use JDBC connections to the data source. Typically
+specific JDBC drivers for the relevant data source are used, and numerous
+extensions in the connector ensure that the specifics of the data sources are
+taken into account. They include variations in data type mappings, support for
+specific SQL keywords and functions, and other aspects.
+
+The following connectors use the JDBC connector base:
+
+* :doc:`clickhouse`
+* :doc:`druid`
+* :doc:`mysql`
+* :doc:`oracle`
+* :doc:`postgresql`
+* :doc:`redshift`
+* :doc:`memsql`
+* :doc:`sqlserver`
+
+These connectors share user-facing behavior and features, documented in the
+following sections.
+
+.. _base-jdbc-configuration:
+
+Configuration
+-------------
+
+.. _base-jdbc-catalog-properties:
+
+Standard catalog properties
```

Comment:
phoenix uses base-jdbc as the base, but has its own configs. how is it going to be modelled here?

Reply:
each connector will link to this document from specific section .. if not applicable.. no link